packages/genesys-spark-components/src/components/beta/gux-side-panel/example.html

```diff
@@ -0,0 +1,143 @@
+<div id="side-panel-wrapper">
+  <div id="content">
+    <h1>Side Panel</h1>
+
+    <p>
+      You can listen to the 'sidePanelDismiss' event on the
+      `gux-side-panel-beta` element to know when the side panel is to be
+      hidden/removed.
+    </p>
+
+    <gux-cta-group>
+      <gux-button slot="secondary" id="toggle-small">Toggle small</gux-button>
+      <gux-button slot="secondary" id="toggle-medium">Toggle medium</gux-button>
+    </gux-cta-group>
```

Comment:
nitpick but this is throwing a console warning due to no primary slot. Also, is the side panel supposed to close when the toggle is pressed twice? It's just opening for me but not closing when I press this.